### ISSUE TITLE
Gperks Warlock

### ISSUE THREAD

**Issue by warius1983** [datchannin/Brothers-WoW#70]
All gperks are working correctly

Suggestion: Shadow destruction; raise power to 10 shadow power

I agree in not using too many special powers to warlock, I beleive raw power is more than enough for him

**Comment by datchannin** [comment]
I agree, I was not able to invent any interesting perks for destruction warlock. Had idea only for affliction.
